cmd/abapEnvironmentCreateSystem.go

```diff
@@ -129,15 +141,53 @@ func generateManifestYAML(config *abapEnvironmentCreateSystemOptions) ([]byte, e
 	return manifestYAML, nil
 }
 
+func checkManifestParameters(config *abapEnvironmentCreateSystemOptions) (err error) {
+
+	//Only checks for correct parameter specification if CF Service "abap-oem" is selected
+	if config.CfService == "abap-oem" {
+
+		if config.AbapSystemParentSaaSAppname == "" && config.AbapSystemParentServiceLabel == "" {
+			return errors.New("Both parameters AbapSystemParentServiceLabel and AbapSystemParentSaasAppname seem to be empty. Please specify either AbapSystemParentServiceLabel or AbapSystemParentSaasAppname depending on who created the oem-instance in the step configuration. For more information please refer to the step documentation")
+		}
+
+		if config.AbapSystemParentSaaSAppname != "" {
+			const parentSaasAppnameSyntaxCheck = `[a-zA-Z0-9\-\_]+`
+			addonProductVersionMatch, _ := regexp.MatchString(parentSaasAppnameSyntaxCheck, config.AbapSystemParentSaaSAppname)
+			if !addonProductVersionMatch {
+				return errors.New("The parameter AbapSystemParentSaaSAppname contains invalid characters. Please check that the parameter follows the respective syntax to specify the AbapSystemParentSaasAppname parameter. For more information please refer to the step documentation")
+			}
+		}
+
+		if config.AbapSystemID != "" {
+			const systemIDSyntaxCheck = `[A-Z0-9]`
+			systemIDMatch, _ := regexp.MatchString(systemIDSyntaxCheck, config.AbapSystemID)
+			if !systemIDMatch {
+				return errors.New("The parameter AbapSystemID contains invalid characters. Please check that the parameter follows the respective syntax to specify the AbapSystemID parameter. For more information please refer to the step documentation")
+			}
+		}
+
+		if config.AbapSystemConsumerTenantLimit == 0 {
```

Comment:
Is this mandatory? @peterpersiel

Reply:
No, parameter is optional and "0" is a valid value
As of now the parameter does not seem to have any meaning
Ask Welf if should be provided, no external use case (?)